db/migrate/20230303213409_add_revalid_to_subjects.rb

```diff
@@ -0,0 +1,5 @@
+class AddRevalidToSubjects < ActiveRecord::Migration[7.0]
+  def change
+    add_column :subjects, :revalid, :boolean, default: false
```

Comment:
We should probably reconsider the naming here...

I couldn't came up with anything so I went with this one instead 🙃 hoping that we would come up with something better later...